pkg/api/helpers.go

```diff
@@ -577,7 +577,7 @@ func SysctlsFromPodAnnotation(annotation string) ([]Sysctl, error) {
 	sysctls := make([]Sysctl, len(kvs))
 	for i, kv := range kvs {
 		cs := strings.Split(kv, "=")
-		if len(cs) != 2 {
+		if len(cs) != 2 || len(cs[0]) == 0 {
```

Comment:
An empty string is a completely valid sysctl value. Unfortunately, we hardly know anything about those values.

During alpha (when we use annotations, we deliberately assume that at least `=` and `,` are no valid characters in values.